strings-csharp/HowToAddSeparatorsInThousandsPlaceForANumber/HowToAddSeparatorsInThousandsPlaceForANumber/NumbersFormatting.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Globalization;
+
+namespace HowToAddSeparatorsInThousandsPlaceForANumber
+{
+    public static class NumbersFormatting
+    {
+        private static readonly double _balance = 2154002.535;
+
+        public static string GetBalanceUsingTheStringFormatMethod()
+        {
+            return String.Format("Your account balance is: {0}", _balance);
```

Comment:
@biagianina 
instead of `String.Format`, let's use `string.Format`

Reply:
Done

Reply:
@biagianina 
This has not been fixed.
The code in `NumberFormatting.cs` still uses `String.Format` instead of `string.format`

Reply:
@elfico It uses `string.Format`, can you check my last commit?